### PR TITLE
Fork fb add transform xml to json using xslt

### PR DESCRIPTION
**Pull request by manuelsaldivar525**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

(FILL ME IN) This section describes why this PR is here. Usually it would include a reference 
to the tracking task that it is part or all of the solution for.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.2.0.4207")]
+[assembly: AssemblyVersion("2.3.0.5157")]
```
Comment (mjkkirschner) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: this file should not be committed.

File: src/Dynamo.All.sln [hunk 2]
```diff
@@ -223,6 +223,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPackagesWPF", "Dynamo
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoWPFCLI", "DynamoWPFCLI\DynamoWPFCLI.csproj", "{F7FD9395-35D5-474D-8AD1-9904817E70D3}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TransformXMLToJSONUsingXSLT", "Tools\TransformXMLToJSONUsingXSLT\TransformXMLToJSONUsingXSLT.csproj", "{676671D5-76B6-4E3B-9244-5323BF6B8EFE}"
+EndProject
```
Comment (mjkkirschner) [inline src/Dynamo.All.sln]: your answer below indicates this does not need to be added to the Dynamo.sln - can you remove it from here?

**Comment by mjkkirschner** [comment]
also note the failing appVeyor build:
```
C:\projects\dynamo\src\Dynamo.All.sln" (default target) (1) ->
(Build target) -> 
  C:\projects\dynamo\src\Dynamo.All.sln.metaproj : error MSB3202: The project file "C:\projects\dynamo\src\Tools\TransformXMLToJSONUsingXSLT\TransformXMLToJSONUsingXSLT.csproj" was not found. [C:\projects\dynamo\src\Dynamo.All.sln]
    2099 Warning(s)
    1 Error(s)
Time Elapsed 00:00:43.94
```

@manuelsaldivar525 I would prefer Dynamo.sln had no dependency on this tool being built.

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs (hunk 1 above)
Comment (manuelsaldivar525) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: Yes, I already added it into git ignore.

File: src/Dynamo.All.sln (hunk 2 above)
Comment (manuelsaldivar525) [reply src/Dynamo.All.sln]: Yes, it will be removed from the main solution.

**Comment by mjkkirschner** [comment]
can this PR be closed? Is there any difference between this and the other PR?

**Comment by manuelsaldivar525** [comment]
> can this PR be closed? Is there any difference between this and the other PR?

Yes, there are some differences but I think it can be close it.

**Pull request closed without merging**